deployment/ccip/changeset/deploy_chain.go

```diff
@@ -0,0 +1,44 @@
+package changeset
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	ccipdeployment "github.com/smartcontractkit/chainlink/deployment/ccip"
+)
+
+var _ deployment.ChangeSet[DeployChainContractsConfig] = DeployChainContracts
+
+func DeployChainContracts(env deployment.Environment, c DeployChainContractsConfig) (deployment.ChangesetOutput, error) {
+	newAddresses := deployment.NewMemoryAddressBook()
+	err := ccipdeployment.DeployChainContractsForChains(env, newAddresses, c.HomeChainSelector, c.ChainSelectors, c.MCMSCfg)
+	if err != nil {
+		env.Logger.Errorw("Failed to deploy CCIP contracts", "err", err, "newAddresses", newAddresses)
+		return deployment.ChangesetOutput{AddressBook: newAddresses}, deployment.MaybeDataErr(err)
+	}
+	return deployment.ChangesetOutput{
+		Proposals:   []timelock.MCMSWithTimelockProposal{},
```

Comment:
Are all of these changesets going to be converted to use MCMS? Or is MCMS for post-deployment processes only (transferring ownership, setting config, etc.)

Reply:
MCMS for post deployment